### ISSUE TITLE
ru locale lua error

### ISSUE THREAD

**Issue by Alex-Miller** [WeakAuras/WeakAuras2#1019]
wa 2.10.6.
wa 2.10.5 was fine.

```
15x WeakAuras\Locales\ruRU-Русский.lua:616: unfinished long string near '<eof>'
Locals:
```

**Comment by wanderkiel** [comment]
![image](https://user-images.githubusercontent.com/42659606/49883658-0d2b9480-fde8-11e8-845a-250d4a6b22a4.png)


Getting the same, the only thing it seems to affect so far is Tehr's CD WA.  It loads a blank bit of the background and none of the info.  Re-importing it fixes it until a reload or relog.

**Comment by InfusOnWoW** [comment]
This was just fixed by the 2.10.7 release
